### PR TITLE
Revoke invites test

### PR DESCRIPTION
**Pull request by ghost**
Adding a test that aims to test that pending invites can be revoked successfully

### CODE REVIEW

File: lib/pages/invite-error-page.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+import By from 'selenium-webdriver';
```
Comment (Stojdza) [inline lib/pages/invite-error-page.js]: It looks like this import should be changed to:
`import { By } from 'selenium-webdriver';`

**Review by Stojdza — CHANGES_REQUESTED** [review]
async/await calls look fine. 

Looking at this error message in CircleCI - `_seleniumWebdriver.default.css is not a function` - it looks like that `import` _in invite-error-page.js_ should be changed.

File: lib/pages/invite-error-page.js (hunk 1 above)
Comment (sofiavistas) [reply lib/pages/invite-error-page.js]: indeed it was this, thanks for the review!

File: specs/wp-invite-users-spec.js [hunk 2]
```diff
@@ -286,6 +271,128 @@ testDescribe( `[${ host }] Invites:  (${ screenSize })`, function() {
 		} );
 	} );
 
+	test.describe.only( 'Inviting New User as an Editor And Revoke Invite: @parallel @jetpack', function() {
```
Comment (Stojdza) [inline specs/wp-invite-users-spec.js]: @sofiavistas Does `.only` should stay here? I do not know will CircleCI skip the other tests from this spec. Maybe somebody can confirm this?

Comment (bsessions85) [reply specs/wp-invite-users-spec.js]: Good catch @Stojdza, yes, CircleCI will skip the other tests, so this should be removed

**Review by Stojdza — APPROVED** [review]

**Comment by alisterscott** [comment]
Thanks for the PR

LGTM so merging now before this gets out of date

**Pull request merged**